recipes-kernel/linux/linux-mainline_5.4.bb

```diff
@@ -1,9 +1,10 @@
 require recipes-kernel/linux/linux-mainline-common.inc
 
-LINUX_VERSION ?= "5.3.x"
+LINUX_VERSION ?= "5.4.x"
 KERNEL_VERSION_SANITY_SKIP="1"
 
-BRANCH = "linux-5.3.y"
+# BRANCH = "linux-5.4.y"
+BRANCH = "master"
```

Comment:
hmmm master seems wrong here

Reply:
Yep, just waiting for the stable branch to be released once 5.4 comes out. Then I'll update this and change the PR from draft state. I just wanted to put this out there for anyone who wants to test.